network/acceptedwork.go

```diff
@@ -35,6 +35,7 @@ type AcceptedWork struct {
 	Height    uint32 `json:"height"`
 	MinedBy   string `json:"minedby"`
 	Miner     string `json:"miner"`
+	CreatedOn uint64 `json:"createdon"`
```

Comment:
I would use int64 instead.

Reply:
it is also, set, but never read anywhere.  is that intended?

Reply:
Yes, the created on time coupled with the logs will be used in analyzing confirmations if there is a bug.